honeycombio/resource_slo_test.go

```diff
@@ -132,9 +132,21 @@ func TestHoneycombSLO_MD(t *testing.T) {
 					resource.TestCheckResourceAttr("honeycombio_slo.md_test", "time_period", "30"),
 				),
 			},
+			// tests imports
+			{
+				ResourceName:      "honeycombio_slo.md_test",
+				ImportState:       true,
+				ImportStateVerify: true,
+				ImportStateIdFunc: func(s *terraform.State) (string, error) {
```

Comment:
as far as I know we haven't needed an `ImportStateIdFunc` previously. Do we need this one? If we do, a comment as to why would be good